src/tokens.scss

```diff
@@ -0,0 +1,4 @@
+:host {
+  --ds-auro-popover-background-color: var(--ds-color-container-primary-default, $ds-color-container-primary-default);
+  --ds-auro-popover-text-color: var(--ds-color-text-primary-default, $ds-color-text-primary-default);
```

Comment:
These are missing the #{...} around the SCSS variable in order to make them compile correctly.

Reply:
Based on already released code, we only need to use `#{...}` when reassigning the value of the tokens in `color.scss`